Add docs for new connectivity test buttons added in Octopus 2019.7.13 (#502)
OctopusDeploy/docs: docs/deployment-process/issue-tracking/jira.md

### Testing the Integration

> **Octopus 2019.7.13** introduced the ability to verify that a connection between the Octopus server and the Jira cloud/Server instance can be made successfully.

Reviewer: ```suggestion
**Octopus 2019.7.13** introduced the ability to verify that a connection can be made successfully between the Octopus server and your Jira Cloud/Server instance.
```

> The **Connect App** `Test` button checks the connectivity for pushing deployment data to a Jira cloud instance. 

Reviewer: ```suggestion
The **Connect App** `Test` button (found under `Jira Connect App Password`), checks the connectivity for pushing deployment data to your Jira Cloud instance. 
```

![Connect App Test button](images/jiraconnectapp_testbutton.png)

:::hint

> For this connectivity test to succeed the Octopus server must be able to connect to https://jiraconnectapp.octopus.com which hosts our Jira plugin and to your Jira cloud instance's Url

Reviewer: ```suggestion
For this connectivity test to succeed the Octopus server must be able to connect to both your Jira Cloud instance's url, and to [https://jiraconnectapp.octopus.com](https://jiraconnectapp.octopus.com), which hosts our Jira plugin.
```

:::

> The **Release Notes** `Test` button checks the connectivity for retrieving work item information from Jira. 

Reviewer: ```suggestion
The **Release Notes** `Test` button (found under `Jira Password`) checks the connectivity to your Jira Cloud / Server instance for retrieving work item information. 
```

![Release Notes Test button](images/jirareleasenotes_testbutton.png)

:::hint

> For this connectivity test to succeed the Octopus server must be able to connect to your Jira cloud/Server instance's Url

Reviewer: ```suggestion
For this connectivity test to succeed the Octopus server must be able to connect to your Jira Cloud/Server instance's url.
```
